cmd/drone-docker/main.go

```diff
@@ -103,6 +103,11 @@ func main() {
 			Usage:  "docker daemon executes in debug mode",
 			EnvVar: "PLUGIN_DEBUG,DOCKER_LAUNCH_DEBUG",
 		},
+		cli.BoolTFlag{
```

Comment:
I think this should be `Bool` instead of `BoolT`.  The `BoolT` parameter defaults to `true` if unset, however, in this case I think we want to disable to false and only disable IP tables if the user explicitly sets this value to `true`

Reply:
or as @tboerger mentioned, you can use `BoolT` to make it true by default, but then below, you need to change from `c.Bool` to `c.BoolT`